eras/conway/impl/src/Cardano/Ledger/Conway/TxCert.hs

```diff
@@ -443,7 +444,8 @@ conwayTxCertDelegDecoder = \case
     pure (3, AuthCommitteeHotKeyTxCert cred key)
   15 -> do
     cred <- decCBOR
-    pure (2, ResignCommitteeColdTxCert cred)
+    a <- decCBOR
```

Comment:
For consistency sake with other anchors lets use null encoding instead. Don't forget to update cddl too ;) 
```suggestion
    a <- decodeNullStrictMaybe decCBOR
```